.travis.yml

```diff
@@ -1,4 +1,4 @@
 language: cpp
-script: ./tools/run_tests/run_tests.py -lc -t -j2
+script: ./tools/run_tests/run_tests.py -lc -lnode -t -j2
```

Comment:
I think the actual command-line is -l c node - the second -lnode is currently overriding the first -lc.